File: VPC.Client/ClientApp/src/app/model/listlayoutdetails.ts
import { SelectedItem } from './selecteditem';
import { OrderDetails } from './orderdetails';
import { SearchProperties } from './searchproperties';
import { RowLevelOperations } from './rowleveloperations';
import { Operation } from './operation';

export class ListLayoutDetails {
  public fields: SelectedItem[];
  defaultSortOrder: OrderDetails;
  maxResult: number;
  searchProperties: SearchProperties[];
  actions: RowLevelOperations[];
  toolbar: Operation[];
  defaultGroupBy : string;
}

File: VPC.Entities/EntityCore/Model/Storage/ListLayoutDetails.cs
using System.Collections.Generic;


namespace VPC.Entities.EntityCore.Model.Storage
{
    public class ListLayoutDetails:ListViewBasicDetails
    {
      
        public int MaxResult { get; set; }
        public List<SearchProperties> SearchProperties { get; set; }
        public List<RowLevelOperations> Actions { get; set; }
        public List<Operation> Toolbar { get; set; }        
        public string DefaultGroupBy { get; set; }
    }
}

